neo/rawio/blackrockrawio.py

```diff
@@ -648,8 +695,9 @@ def _event_count(self, block_index, seg_index, event_channel_index):
 
     def _get_event_timestamps(self, block_index, seg_index, event_channel_index, t_start, t_stop):
         name = self.header['event_channels']['name'][event_channel_index]
-        events_data = self.nev_data['NonNeural']
+        events_data, event_segment_ids = self.nev_data['NonNeural']
         ev_dict = self.__nonneural_evtypes[self.__nev_spec](events_data)[name]
+        mask = ev_dict['mask'] & (event_segment_ids == seg_index)
 
         timestamp = events_data[ev_dict['mask']]['timestamp']
         labels = events_data[ev_dict['mask']][ev_dict['field']]
```

Comment:
mask should be used here instead of ev_dict['mask'] for the 2 previsous lines

Reply:
Yes, you are right. I fixed this.